### PR TITLE
Feature/3524 score card example

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes # BLUI-3524

#### Changes proposed in this Pull Request:
- Added examples for Score card

#### Screenshots:
Screenshot 2022-09-27 at 4 53 25 PM

#### To Test:
- yarn start:reactdev

### CODE REVIEW

**Comment by emclaug2** [comment]
These examples look very nice; one small comment - 

Score Card w/ Heroes highlight needs adjusted. 
image

File: docs/src/componentDocs/ScoreCard/markdown/ScoreCardExamples.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# Score Card Header with Subtitle and Info
+
+The `<ScoreCard>` also accepts a `headerSubtitle` and `headerInfo` props to provide additional context.
+
+<ScoreCardWithHeaderSubtitle />
+
+# Score Card with Actions
+
+The `<ScoreCard>` accepts a `actionItems` and `actionRow` props to allow users to interact with each card.
```
Comment (huayunh) [inline docs/src/componentDocs/ScoreCard/markdown/ScoreCardExamples.mdx:21]:
```suggestion
The `<ScoreCard>` accepts an `actionItems` and `actionRow` props to allow users to interact with each card.
```

File: docs/src/componentDocs/ScoreCard/examples/ScoreCardWithScoreBadge.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            <InfoListItem dense chevron title="View Location" hidePadding />
+        </List>
+    }
+    badge={
+        <HeroBanner>
+            <Hero
+                icon={<GradeA fontSize={'inherit'} />}
+                label={'Grade'}
+                iconSize={72}
+                iconBackgroundColor={colors.white[50]}
```
Comment (huayunh) [inline docs/src/componentDocs/ScoreCard/examples/ScoreCardWithScoreBadge.tsx:23]:
```suggestion
                iconBackgroundColor={'background.paper'}
```

Comment (huayunh) [reply docs/src/componentDocs/ScoreCard/examples/ScoreCardWithScoreBadge.tsx:23]: (I don't know if our component can actually take these MUI v5 inputs... please ignore this comment if things breaks

**Review by huayunh — COMMENTED** [review]
what Evan said above, plus:

**Comment by emclaug2** [comment]
It still looks like the code highlighting for Score Card with Heroes is off by 1 line.

**Comment by bkarambe** [comment]
> It still looks like the code highlighting for Score Card with Heroes is off by 1 line.

@emclaug2 Updated

**Review by jeffvg — APPROVED** [review]
looks good, reviewed items updated

**Review by emclaug2 — APPROVED** [review]

**Pull request merged**